### PR TITLE
feat: Add WinUI3 support

### PR DESCRIPTION
**Pull request by ebariche**

### CODE REVIEW

File: src/library/Uno.Cupertino/CupertinoResources.cs [hunk 1]
```diff
@@ -30,7 +35,9 @@ public CupertinoResources()
 			this.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri("ms-appx:///Uno.Cupertino/Styles/Controls/NumberBox.xaml") });
 			this.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri("ms-appx:///Uno.Cupertino/Styles/Controls/PasswordBox.xaml") });
 			this.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri("ms-appx:///Uno.Cupertino/Styles/Controls/ProgressBar.xaml") });
+#if !WinUI_Desktop
```
Comment (jeromelaban) [inline src/library/Uno.Cupertino/CupertinoResources.cs:38]: Let's add an issue in Uno and reference it here

File: src/library/Uno.Material/MaterialResources.cs [hunk 2]
```diff
@@ -33,10 +38,14 @@ public MaterialResources()
 			this.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri("ms-appx:///Uno.Material/Styles/Controls/Flyout.xaml") });
 			this.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri("ms-appx:///Uno.Material/Styles/Controls/HyperlinkButton.xaml") });
 			this.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri("ms-appx:///Uno.Material/Styles/Controls/InfoBar.xaml") });
+#if !WinUI
```
Comment (jeromelaban) [inline src/library/Uno.Material/MaterialResources.cs:41]: Same here for the issue link

File: src/library/Uno.Material/Uno.Material.csproj [hunk 3]
```diff
@@ -1,51 +1,106 @@
...
+				<DefineConstants Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'">$(DefineConstants);WinUI_Desktop</DefineConstants>
+				<PackageId>Uno.Material.WinUI</PackageId>
+				<TargetFrameworks>$(TargetFrameworks);net5.0-windows10.0.18362</TargetFrameworks>
+			</PropertyGroup>
+			
+			<ItemGroup>
+				<PackageReference Include="Uno.WinUI" Version="3.7.0-dev.251" />
+
+				<PackageReference Include="Uno.WinUI.Lottie"
+													Version="3.7.0-dev.251"
```
Comment (jeromelaban) [inline src/library/Uno.Material/Uno.Material.csproj:26]: no need to indent here

**Review by jeromelaban — APPROVED** [review]

File: src/library/Uno.Material/Uno.Material.csproj (hunk 3 above)
Comment (ebariche) [reply src/library/Uno.Material/Uno.Material.csproj:26]: If we were using spaces everywhere...

**Pull request merged**